### PR TITLE
Preparation of release 3.6.9

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ ergw-aaa
 
 Erlang AAA session implementation for ERGW.
 
+Version 3.6.9 - 4 February 2021
+----------------------------
+
+**Features** :rocket:
+* [#116](https://github.com/travelping/ergw_aaa/pull/116) Adding the `DIAMETER` `RFC` base dictionary to all services
```
Comment (mgumz) [inline NEWS.md:10]: what does this yield in terms of features?

File: NEWS.md [hunk 2]
```diff
@@ -3,6 +3,15 @@ ergw-aaa
...
+
+**Dependencies** :gear:
+* [#117](https://github.com/travelping/ergw_aaa/pull/117) Update [eradius](https://github.com/travelping/eradius) tag to [2.2.1](https://github.com/travelping/eradius/releases/tag/2.2.1)
```
Comment (mgumz) [inline NEWS.md:13]: In short: minor fixes and features?

Comment (vkatsuba) [reply NEWS.md:13]: This is not a features this is **Dependencies** group. And if we update dependencies we put them into **Dependencies** section.

File: NEWS.md (hunk 1 above)
Comment (vkatsuba) [reply NEWS.md:10]: Looks like this is **Features** by reason that - we was not have `DIAMETER` `RFC` base in project before, if we add something new this is **features**, if we change something for eg speed up reload something etc. - this is **improvements**. Plus for current implementation was created branch with prefix `feature`.

**Pull request merged**